Description: Corrects the incorrect bit-shift in pntohl(), the left-shift should be 8 bits, not 18.
Author: Matthew Boyle <mlb@decisionsoft.co.uk>
Origin: http://pkgs.fedoraproject.org/cgit/rpms/dsniff.git/tree/dsniff-2.4-pntohl_shift.patch
Bug-RedHat: https://bugzilla.redhat.com/show_bug.cgi?id=714958
Bug-RedHat: https://bugzilla.redhat.com/show_bug.cgi?id=850496
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/decode.h
+++ b/decode.h
@@ -35,7 +35,7 @@
 			  (u_short)*((u_char *)p+0)<<8))
 			 
 #define pntohl(p)	((u_int32_t)*((u_char *)p+3)<<0|   \
-			 (u_int32_t)*((u_char *)p+2)<<18|  \
+			 (u_int32_t)*((u_char *)p+2)<<8|  \
 			 (u_int32_t)*((u_char *)p+1)<<16|  \
 			 (u_int32_t)*((u_char *)p+0)<<24)
 
